lib/models/addon.js

```diff
@@ -1239,7 +1243,7 @@ let addonProto = {
     return new Funnel(lintedAddon, {
       srcDir: '/',
       destDir: `${this.name}/tests/`,
-      annotation: `Funnel: Addon#jshintAddonTree(${this.name})`,
+      annotation: `Funnel: Addon#lintAddonTree(${this.name})`,
```

Comment:
I am not sure if stats are depending on annotations, so the question is that do we need a long lead time to change the annotation.